### DISCUSSION TITLE
Spawn only friendly mobs

### DISCUSSION THREAD

**Discussion by lockf** [Faboslav/friends-and-foes#131] [Q&A] [answered]
Is it possible to use this mod but have it just spawn the selected mobs like copper golem, glare, Tuff Golem, ... and not spawn enemy mobs on any dificulty ?

-lockf

**Comment by Faboslav** [comment, marked as answer]
Hello, sorry i always forget about this section on github, you can customize(read enable/disable) spawns via the config with modmenu or in json file.

**Reply by lockf** [reply]
ty
